common-libs/internal-api-clients/src/services/RegistryApiService.ts

```diff
@@ -24,7 +24,16 @@ export default class RegistryApiService {
   }
 
   async anchorDid(params: GetParams<typeof clientMethods.AnchorDid>) {
-    return this.client.AnchorDid({ params })
+    const originFromEnv = process.env.REGISTRY_ORIGIN
```

Comment:
@dhuan-affinidi  do we have a sdk option for origin ? why we have a need for env var on core dsk code? it could be a challenge for react native users.